src/page-data/may-2020/amp.js

```diff
@@ -62,7 +62,7 @@ export const data = {
     },
     'fcpCrux': {
       numOrigins: 5632,
-      data: [34, 20, 46],
+      data: [44, 28, 28],
```

Comment:
Did you run the query for May and notice this error? Thanks for spotting + fixing!!

Reply:
Or is it a mistake? Because it looks the same as june-2020 🤔

Reply:
I checked the data for May, 2020 and June, 2020 and it's similar in case of FCP